tests/test/Counter.t.sol

```diff
@@ -1,30 +1,32 @@
 // SPDX-License-Identifier: AGPL-3.0
 pragma solidity >=0.8.0 <0.9.0;
 
+// NOTE: required options: --loop 4 --symbolic-storage
```

Comment:
out of scope for this but it would be really great if we could annotate it at the contract or method level

```
/// @custom:halmos loop=4
/// @custom:halmos symbolic-storage=true
```

Reply:
yes, it would be great!  it's on the todo list.